localstack/services/events/rule.py

```diff
@@ -0,0 +1,91 @@
+import re
+from typing import Optional
+
+from localstack.aws.api.events import (
+    Arn,
+    EventBusName,
+    EventPattern,
+    RoleArn,
+    RuleDescription,
+    RuleName,
+    RuleState,
+    ScheduleExpression,
+    TagList,
+)
+from localstack.services.events.models_v2 import Rule, ValidationException
+
+
+class RuleWorker:
```

Comment:
sidenote: Worker indicates some kind of running process. Maybe that's gonna be the case one the threading functionality comes and we can see later whether the name still fits ...

Reply:
renamed to service in: https://github.com/localstack/localstack/pull/10613/commits/71ab174d6154a4b7848c1dc1486850e9a541da8d